packages/bsky/src/api/com/atproto/identity/resolveHandle.ts

```diff
@@ -13,6 +13,16 @@ export default function (server: Server, ctx: AppContext) {
     if (user) {
       did = user.did
     } else {
+      const publicHostname = ctx.cfg.publicUrl
```

Comment:
we just gotta ensure we don't have any alternate subdomains for the appview 😛

Reply:
Yeah 😳 I wonder if we should eventually wrap this into a more general configuration allowing multiple hostnames or even wildcards.  I'll leave a note here for now.